index.html

```diff
@@ -395,30 +409,34 @@ <h3 class="subtitle is-5">Don't hesitate, ask anything</h3>
             <fieldset class="field">
               <label class="label" for="username">Username</label>
               <div class="control has-icons-left has-icons-right">
-                <input id="username" class="input is-success" type="text" name="username">
-                <span class="icon is-small is-left">
+                <input id="username" class="input is-success" type="text" name="username"
+                  placeholder="Enter your username" aria-describedby="usernameHelp"
+                  aria-invalid="false">
+                <span class="icon is-small is-left" aria-hidden="true">
                   <i class="fas fa-user"></i>
                 </span>
-                <span class="icon is-small is-right">
+                <span class="icon is-small is-right" aria-hidden="true">
                   <i class="fas fa-check"></i>
                 </span>
               </div>
-              <p class="help is-success">This username is available</p>
+              <p id="usernameHelp" role="alert" class="help is-success">This username is
+                available</p>
             </fieldset>
 
             <fieldset class="field">
               <label class="label" for="email">Email</label>
               <div class="control has-icons-left has-icons-right">
                 <input id="email" class="input is-danger" type="email"
-                  placeholder="Enter your email" name="email" value="hello@">
-                <span class="icon is-small is-left">
+                  placeholder="Enter your email" name="email" value="hello@"
+                  aria-describedby="emailHelp" aria-invalid="true">
```

Comment:
You can also add `aria-required=true` for all obligatory controls within this form. It would be beneficial for users of screen readers as it explicitly indicates that a field is required. But this is only if we talk about current hometask purpose. The preferable option is to add a `required` attribute for form elements. For browsers and assistive technologies that support HTML5 the required attribute should serve the same purpose as `aria-required=true` and tell users with assistive technologies additional information about the form